teemaderegister-react/.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{    
```

Comment:
this complete file could be removed or please comment if debugger actually works?